plugins/json-schema/README.md

```diff
@@ -1,7 +1,7 @@
-# @flatfile/plugin-convert-json-schema
+# @flatfile/plugin-convert-yaml-schema
```

Comment:
The changes to the json-schema folder need to be reverted

Reply:
Yeah, the changes seem to have gotten mixed up at some point. They should be good now though